automatic/kubescape/kubescape.nuspec

```
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>kubescape</id>
+    <version>2.3.3</version>
+    <packageSourceUrl>https://github.com/chocolatey-community/chocolatey-packages/tree/master/automatic/kubescape</packageSourceUrl>
+    <owners>chocolatey-community</owners>
```

```suggestion
    <owners>chocolatey-community,hollowman</owners>
```

The owner section should include any historical and current owners of the package.